parts/language/manual_ja.txt

```diff
@@ -13,50 +13,103 @@
 スピンの検出:
         「3コーナ」ルールを満たした時 +2点
         「インモービル (Immobile)」 ルールを満たした時 +2点
-        - 上のどちらかを満たした時, スピンが確定します。
+        - 上のどちらかを満たした時、スピンが確定します。
 
-        もし回転システムがキックテーブルから2番チェック (2nd check) を採用していなかった場合 +1点
-        - もしスピンの合計点が2点しかない場合は「ミニ」扱いになります；ミノがS, Z, J, L, Tのいずれかで, ラインクリア時にそのミノを全て消せなかった場合のみ。
+        もし回転システムがキックテーブルから2番目のチェック (2nd check) を採用していなかった場合 +1点
+        - もしスピンの合計点が2点しかない場合は「ミニ」扱いになります。ただしこれは、ミノがS,Z,J,L,Tのいずれかで、line消去の時にそのミノのブロックが全部消せなかった場合のみ。
 
-Attack system:
-        [TODO]
-Back to Back (B2B) gauge:
-        [TODO]
+攻撃システム:
+        Special clear:
+                Special clearとは、spin、techrash(もしくは一度に4段以上のline消去)、PC、HPCのどれかを行うことです。Special clearを行うとB2Bゲージが上昇します。
+        Super clear:
+                Super clearとは、B2Bゲージがある状態でSpecial clearを行うことです。
+        Special clears (spin):
+                消したline数の2倍の火力を送ります、
+                - B2Bは、spin single/doubleで1line、spin tripleで2line、spin techrashで4line、spin techrash+で8lineを火力に加算します。
+                - B2B2Bは、消したline数の半分を火力に加算し、更に+1lineを相殺専用火力に加算します。
+                - またspinの場合は、B2BとB2B2Bは両方付与されます。
+                - ミニは、火力を25%に減少させます。(倍率0.25倍)
+
+        Special clears (spin以外のtechrash/techrash+):
+                - B2Bは、火力に+1lineします。
+                - B2B2Bは、火力を50%上昇させ、更に+1lineを相殺専用火力に加算します。
+        
+        Non-special clears:
+                火力-0.5段を送ります。
+
+        ren:
+                ren数は、連続してline消去した回数-1回です。PCとHPCを除くSupecial clearにren数*25%(1+ren*0.25)、Non-special clearにren数*15%(1+ren*0.15、最大12*15%)を乗算します。
```

Comment:
"except attack from PC/HPC" is redundant even in the English version. It was there because PC/HPC were written above combo before.

Reply:
`capped at 12 combos` means `ren <= 12` when calculating.
For example, attack of `double 14 combo` is `(2-0.5) * (1 + 12 * 0.25) = 6` (notice 12 instead of 14 in the calculation); attack of `single 14 combo` is `(1-0.5) * (1 + 12 * 0.15) = 1.4`; attack of `s-spin single 14 combo` is `2 * (1 + 12 * 0.15) = 5.6`.

Reply:
Do you mean you are judging whether it is 1-line or 2-line or more?